lib/msf/core/option_container.rb

```diff
@@ -728,6 +728,18 @@ def validate(datastore)
     return true
   end
 
+  def check_warnings(datastore)
+    warnings = []
+
+    # message about missing SSL
+    if datastore.include?('RPORT') && [443, 8443].include?(datastore['RPORT']) &&
```

Comment:
Probably you would like to make an extra method for this check. So if there are more checks in the future, they can ad their own method, instead of making this method bigger and bigger.